playwright.config.ts

```diff
@@ -101,6 +102,7 @@ const config: PlaywrightTestConfig = {
       },
       use: {
         ...devices["Desktop Chrome"],
+        locale: "en-US",
```

Comment:
My system is in Spanish, added this so tests are consistent regardless system locale.